src/tests/JIT/Regression/JitBlue/Runtime_39424/Runtime_39424.il

```diff
@@ -0,0 +1,196 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+
+// The test was showing some rare cases of `IND(LCL_FLD_ADDR)` under HW intrinsic nodes.
+
+.assembly extern System.Runtime {}
+.assembly extern System.Runtime.Intrinsics {}
+.assembly extern System.Runtime.Extensions {}
+.assembly extern System.Numerics.Vectors {}
+.assembly extern System.Numerics.Extensions {}
+.assembly extern System.Runtime.CompilerServices.Unsafe {}
+
+.assembly Runtime_39424.exe {}
+
+.class private auto ansi beforefieldinit Runtime_39424
+       extends [System.Runtime]System.Object
+{
+  .class sequential ansi sealed nested private beforefieldinit Container
+         extends [System.Runtime]System.ValueType
+  {
+    .field public valuetype [System.Numerics.Vectors]System.Numerics.Vector`1<int32> Vector
+    .field public float64 Double
+  } // end of class Container
+
+  .method private hidebysig static float64 
+          TestLclFldAddrIntrinsicsRound(float64 d) cil managed noinlining
+  {
+    // Code size       30 (0x1e)
+    .maxstack  2
+    .locals init (valuetype Runtime_39424/Container V_0)
+    IL_0000:  ldloca.s   V_0
+    IL_0002:  initobj    Runtime_39424/Container
+    IL_0008:  ldloca.s   V_0
+    IL_000a:  ldarg.0
+    IL_000b:  stfld      float64 Runtime_39424/Container::Double
+    IL_0010:  ldloca.s   V_0
```

Comment:
```
ldloca.s   V_0 
ldflda float64 Runtime_39424/Container::Double
volatile. ldind.r8
```
were generating what we needed.  I have disabled CSE, added more than 4 fields to avoid struct promotion and marked indir as volatile to avoid the optimizations that were hiding the issue.